mkdocs.yml

```diff
@@ -248,6 +250,7 @@ plugins:
         'plugins/porting-from-netbox.md': 'development/apps/porting-from-netbox.md'
         'rest-api/overview.md': 'user-guide/platform-functionality/rest-api/overview.md'
         'user-guide/administration/configuration/node-configuration.md': 'user-guide/administration/configuration/index.md'
+        'user-guides/contact-and-team.md': 'user-guide/feature-guides/contact-and-team.md'
```

Comment:
Don't really need a redirect here since this isn't a pre-existing documentation page.